config/locales/global/en.yml

```diff
@@ -35,14 +35,14 @@ en:
         icon: icons/cold-coral.svg
     nav:
       burger_text: Explore a country/region
-    
+
     footer:
       about: About Ocean+ Habitats
       legal: Legal
-      copyright: Global Ocean Habitat Profile from the Ocean+ &copy; copyright
+      copyright: '&copy; 2021 UN ENVIRONMENT PROGRAMME (UNEP)'
```

Comment:
Can we add the 2021 with Date.today.year in the rails view using the i18n interpolation? Currently you have 2021 twice!